### DISCUSSION TITLE
ОШИБКА КОМПИЛЯЦИИ В ВЕСИИ V1.13

### DISCUSSION THREAD

**Discussion by sergejsimak** [vvip-68/GyverPanelWiFi#319] [Общие вопросы]
![ОШИБКА  КОМПИЛЯЦИИ](https://github.com/user-attachments/assets/010f6bef-9bc5-44d2-9ae8-762b135055b3)
ОШИБКА КОМПИЛЯЦИИ В ВЕСИИ V1.13
ПОМОГИТЕ ПОЖАЛУЙСТА
![ОШИБКА  КОМПИЛЯЦИИ](https://github.com/user-attachments/assets/7dc46eaa-4832-4c2f-8171-bb69531ae3cb)

**Comment by vvip-68** [comment]
![image](https://github.com/user-attachments/assets/dfd9b1ed-7f6b-4a54-b1da-15f9b01502d8)
